### ISSUE TITLE
Making a movie genre classification model? I have different genres and I have vectorized them? What should be the shape of my label tensor if my data tensor is 2d?

### ISSUE THREAD

**Issue by Ahzam-Lathiya** [keras-team/keras#13152]
The [keras.preprocessing](https://github.com/keras-team/keras-preprocessing) module has been splitted from the main Keras repository.

Please submit your issue using this [link](https://github.com/keras-team/keras-preprocessing/issues/new/choose).

Thank you!

**Comment by Ahzam-Lathiya** [comment]
I have 21 movie genres as my categorized labels.
I've vectorized my labels with to_categorical() function and it has given me a tensor of shape (1, 21, 21).
WHile my data tensor has shape(15000, 1000).
